src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -61,13 +63,23 @@ public static void setUp() throws IOException, InterruptedException {
      */
     @AfterAll
     public static void tearDown() {
-        assumeTrue(isEndToEnd);
-        setAuthDisabled(getDefaultAuthDisabled());
-        // Delete admin user.
-        Persistence.adminUsers.removeById(adminUser.id);
+        assumeTrue(isEndToEndAndAuthIsDisabled());
```

Comment:
Using something like `isEndToEndAndAuthIsDisabled` no longer makes sense here. `authDisabled` is now set manually in this test class' `setUp` method.

Reply:
Updated.